Adyen/UI/List/ListFooterView.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+internal final class ListFooterView: UIView {
+
+    /// The list section header style.
+    internal let style: ListSectionFooterStyle
+
+    internal init(title: String, style: ListSectionFooterStyle) {
+        self.title = title
+        self.style = style
+
+        super.init(frame: .zero)
+
+        backgroundColor = style.backgroundColor
+        addSubview(stackView)
+
+        stackView.adyen.anchor(inside: self)
+    }
+
+    @available(*, unavailable)
+    internal required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Title Label
+
+    private let title: String
+
+    // MARK: - UI
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [separatorView, titleContainerView])
+        stackView.axis = .vertical
+        stackView.alignment = .fill
```

Comment:
I belive `alignment = .fill` and `distribution = .fill` is default

Reply:
being explicit is better IMO